website/landingpage/dev/src/assets/js/app.js

```diff
@@ -153,11 +153,43 @@ function removeActiveState(arr) {
 }
 
 // BLOG
+var backToBlog = $('#back-to-blog');
+//variables for load more blogs functionality
+var loadMoreBlogsBtn = $('#load-more-blogs-btn');
+var backToTopBtn = $('#back-to-top-btn');
+var filesAmount = $('#blog-chunks-data').data("chunk-total"); //count amount of all blogs files
```

Comment:
One suggestion:
if (filesAmount === undefined) {
  loadMoreBlogsBtn.hide();
}
Probably this will not happen, but if it will happen :)